ansible/tasks/profile/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Configure PAM to create $HOME if it doesn't exist
+  lineinfile:
```

Comment:
instead, use `pam-auth-update --package --enable mkhomedir`

there doesn't appear to be an Ansible module for this, unfortunately.